deliverables/README.md

```diff
@@ -1,8 +1,10 @@
 ##Week 2: Introduction to Ruby
 ###Intro to the Back End
 ####Required
-What is the difference between front- and back-end development?<br>
+What is the difference between front- and back-end development?<br>  
+Front-end development deals with the parts of the webisite that you can see and interact with, and backend development stores and organizes the underlying data.  It uses an application to interface with the frontend, but it resides on a server. 
 Why do you need to install Ruby but not HTML/CSS?
+There are multiple backend development languages so you have to tell your computer that you are going to create applications using Ruby.  HTML and CSS are universally recognized.  The coding output is rendered as a web page and will appear as long as you have a web interface and bandwidth.
```

Comment:
Not quite. Ruby is an interpreted language which means your computer has to run it, which is only done with Ruby installed. With HTML and CSS, your browser is able to understand them directly. 

See this answer: http://stackoverflow.com/questions/7284179/is-ruby-a-scripting-language-or-an-interpreted-language?answertab=active#tab-top